recipes-domd/agl/files/meta-xt-prod-extra/inc/agl-image.inc

```diff
@@ -2,6 +2,7 @@ IMAGE_INSTALL_append = " \
     guest-addons \
     guest-addons-bridge-up-notification-service \
     guest-addons-bridge-config \
+    guest-addons-xendriverdomain-service \
```

Comment:
"update guest addons" patches contain so many unrelated changes that I would love them to be split...

Reply:
Done